policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/solidarity/housing_component/child_amount.yaml

```diff
@@ -0,0 +1,14 @@
+description: Quebec adds the housing component of the solidarity tax credit for each eligible dependent at this amount.
+values:
+  2022-07-01: 128
+metadata:
+  unit: currency-CAD
+  label: Quebec Solidarity Tax Credit QST housing component child amount
+  reference:
+    - title: Revenu Quebec - Solidarity Tax Credit - Housing Component
+      href: https://www.revenuquebec.ca/en/citizens/tax-credits/solidarity-tax-credit/components-of-the-solidarity-tax-credit/housing-component/
+    - title: SPSD/M - 1.3.25.4.10.8 QSTCHCQuebec Solidarity Tax Credit housing component child amount (Page 1650)
+    - title: Parameters of the Personal Income Tax System for 2022 - Table 4
+      href: http://www.finances.gouv.qc.ca/MAJ2021/documents/en/AUTEN_IncomeTax2022.pdf
+    - title: Taxation Act(CQLR c I-3), Part I, Book IX, Title III, Division II.17.2 — Solidarity Credit [1029.8.116.12 - 1029.8.116.35]
```

Comment:
Specific subsection please

Reply:
Fixed! One more concern that there is no maximum family income limit in the Act, which is differ from the Quebec website. 

I also calculated that the credit is 0 when the maximum income is reached. So I kept this eligibility, but did not find a legal reference for that.